io/driver/module_unload_load.sh

```diff
@@ -16,7 +16,11 @@
 
 CONFIG_FILE="$AVOCADO_TEST_DATADIR"/config
 BUILT_IN_DRIVERS=`cat /lib/modules/$(uname -r)/modules.builtin |awk -F"/" '{print $NF}'|sed 's/\.ko//g'`
-DRIVERS=`find /lib/modules/$(uname -r)/ -name \*.ko | awk -F"/" '{print $NF}'|sed 's/\.ko//g'`
+if [[ $ONLY_IO == True ]]; then
```

Comment:
oh my bad luck it should be like this way 
```
**ONLY_IO=true
if $ONLY_IO; then** 
```
example :
```
[praveen@dhcppc6 ~]$ ONLY_IO=true
[praveen@dhcppc6 ~]$ if $ONLY_IO; then echo "Test"; fi 
Test
[praveen@dhcppc6 ~]$ ONLY_IO=false
[praveen@dhcppc6 ~]$ if $ONLY_IO; then echo "Test"; fi 
[praveen@dhcppc6 ~]$ 
```

i think we need to change yaml file as well